### ISSUE TITLE
`bpy_prop_collection` missing `__iter__`

### ISSUE THREAD

**Issue by MicaelJarniac** [nutti/fake-bpy-module#114]
Related to #97.

```py
for window in bpy.context.window_manager.windows:
    # "bpy_prop_collection[Window]" has no attribute "__iter__" (not iterable) | mypy(error)
    pass
```

On the real `bpy`, it does seem to have `__iter__` (see issue #97).

I'm actually not too sure what's the issue here since, for some `for`s, Mypy isn't complaining.

**Comment by nutti** [comment]
@MicaelJarniac 

At that moment, I added `__iter__`  method to `bpy_prop_collection` when I fixed the issue #97 .
However, adding `__iter__` method does not work for my environment (VSCode).
Does it work on your environment?

**Comment by nutti** [comment]
It seems that my environment was bad.
Fixed version works correctly in my environment.
